src/tests/ftest/interoperability/updown_grade.py

```diff
@@ -1,626 +1,71 @@
-'''
-  (C) Copyright 2022-2023 Intel Corporation.
+"""
```

Comment:
What's the difference between `down_grade.py` and `updown_grade.py`? They look to have the same config, and the only difference is that `updown_grade.py` has a variant that passes an extra argument: `self.upgrade_and_downgrade(fault_on_pool_upgrade=True)`
Is `down_grade.py` a duplicate?

Reply:
Thanks for the comments, there is no duplication..
1. updown_grade.py is starting with an old-version (rpms), and perform upgrade and then downgrade test.
2. down_grade.py is starting with a formatted new-version (rpms) and perform downgrade test.
3. Above are positive testcases, there are negative tests of pool-upgrade (with fault_on_pool_upgrade=True)

Reply:
I have the same question. How can the behavior of the two tests differ if they both call the same method (`self.upgrade_and_downgrade()`) with the same yaml? Do we need to prepare different versions of RPMs somewhere before running the tests? (These are manual tests, so I believe the assumptions may be different from the standard avocado tests.)